bn-models/src/models/_lookups/non-flow-scale-reason.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseLookup } from '../_base';
+
+export const nonFlowScaleReasonCode = 'non-flow-scale-reason';
+
+export interface nonFlowScaleReasonCode extends BaseLookup {
+    reason: string
```

Comment:
could this be stored in the BaseLookup.description field as in other lookups?

Reply:
yes it could be, i'll remove reason then and update the docs in couch